source/_docs/getting-support.md

```diff
@@ -5,63 +5,38 @@ keywords: support, scope, customer support, help
 categories: [managing]
 tags: [platform]
 ---
-Submit a ticket with our Support Team to request assistance on issues within our [scope of support](/docs/scope-of-support/).
+Chat with our Support Team to request assistance on issues within our [scope of support](/docs/scope-of-support/).

```

Comment:
We should link to the support tab in the User Dashboard here or provide clarification on how users initiate a support chat.